app/models/offer.js

```diff
@@ -1,85 +1,125 @@
-import Ember from 'ember';
-import DS from 'ember-data';
+import Ember from "ember";
+import DS from "ember-data";
 
```

Comment:
added this helper, it creates a computed prop with the following rules :
- if records are loaded, we count them
- otherwise, we return the value of the 'counter property' that the api returns

Reply:
this comment refers to `createCounterOf`